src/__tests__/components/__snapshots__/Routes.test.js.snap

```diff
@@ -8,7 +8,7 @@ exports[`Routes Component should match snapshot 1`] = `
       "getState": [Function],
       "replaceReducer": [Function],
       "subscribe": [Function],
-      Symbol(observable): [Function],
+      Symbol(Symbol.observable): [Function],
```

Comment:
What's this line doing?

Reply:
This is a snapshot test, from a previous branch

Reply:
I think snapshots should be in your `.gitignore`. Not sure though.

Reply:
okay.